src/backend/flags.rs

```diff
@@ -0,0 +1,76 @@
+use std::ops::{BitOr, BitOrAssign};
+
+#[derive(Copy, Clone, Debug)]
+/// A flag accepted by the `seccomp` syscall.
+pub enum SeccompFlag {
```

Comment:
I don't see the reason for creating such a wrapper over the libc flags.
I suggest simply accepting a `u64` in the `apply_filter_with_flags` function. The user is then responsible for setting their own flags if they so wish.